packages/xod-client/src/editor/containers/Editor.jsx

```diff
@@ -212,24 +212,26 @@ class Editor extends React.Component {
       null,
       R.ifElse(
         R.propEq('id', DEBUGGER_TAB_ID),
-        () => <Breadcrumbs />,
+        () => <Breadcrumbs>
```

Comment:
Overcomplication/overnesting detected. Instead of `ifElse`, would a simple ternary read better?

```js
const maybeBreadcrumbs = R.chain(
  tab => (tab.id === DEBUGGER_TAB_ID && this.props.isDebugSessionRunning) ?
    Maybe.of(
      <button
        className="debug-session-stop-button Button Button--light"
        onClick={this.props.stopDebuggerSession}
      >
      <Icon name="stop" /> Stop debug
      </button>
    ) : Maybe.Nothing()
  this.props.currentTab
);

const breadcrumbs = foldMaybe(null, R.identity, maybeBreadcrumbs);
```